utils/utils.js

```diff
@@ -119,6 +111,28 @@ async function sendTree(gitTree, currentCommitSha, repo, accessToken, message, b
   });
 }
 
+// send the new tree object back to Github and point the latest commit on the staging branch to it
```

Comment:
Can we update the comment? We're not sending the tree object back

Reply:
Whoops, fixed in https://github.com/isomerpages/isomercms-backend/pull/91/commits/8983a82c26aaa8552d21142f108a82364d20bf4f